tests/integ/modin/frame/test_assign.py

```diff
@@ -17,7 +17,7 @@
 )
 
 
-@sql_count_checker(query_count=2, join_count=1)
+@sql_count_checker(query_count=1, join_count=2)
```

Comment:
I see a lot of tests here where query count has gone down but join count has gone up, could you explain what exactly the query being removed was?

Reply:
It's the way the DataFrame/Series is being constructed - previously a query was being issued to convert the index to a native pandas object when creating the Series/DataFrame with lazy index, now a join is performed with the lazy index instead (query compiler generated for native pandas joins with the lazy index).